src/fleet/azext_fleet/_help.py

```diff
@@ -23,6 +23,30 @@
           type: string
           short-summary: Prefix for hostnames that are created. If not specified, generate a hostname using the
                          managed cluster and resource group names.
+        - name: --enable-private-cluster
+          type: bool
+          short-summary: Whether to create the Fleet hub as a private cluster or not.
+        - name: --enable-vnet-integration
+          type: bool
+          short-summary: Whether to enable apiserver vnet integration for the Fleet hub or not.
+        - name: --apiserver-subnet-id
+          type: string
+          short-summary: he subnet to be used when apiserver vnet integration is enabled. It is required when creating a new Fleet with BYO vnet.
+        - name: --agent-subnet-id
+          type: string
+          short-summary: The ID of the subnet which the Fleet hub node will join on startup. If this is not specified, a vnet and subnet will be generated and used.
+        - name: --identity-type
+          type: string
+          short-summary: Type of managed service identity (where both SystemAssigned and UserAssigned types are allowed).
```

Comment:
If it's a user assigned identity, the user needs to provide the resource ID of the identity somehow.
If it's a system assigned identity, the user just says "give me a system assigned identity".

Reply:
This is the property to state which identity type is set. As you stated it's either User Assigned or System Assigned.  This summary text is from https://github.com/Azure/azure-rest-api-specs/blob/def187e2e78d7173d8fdd7f77740dd9719e1dfbf/specification/common-types/resource-management/v4/managedidentity.json#L36. 

What I believe you are saying is that we should change the params to take in the RID of the user assigned identity only. The fields TID / Principal ID, Object ID, and type are will be filled by the RP (us) and only readonly from a client side. 

Is that correct?

Reply:
this is completely standard cli functionality.
did you take example on how other cli do this? (param name, types, docs, etc..)

Reply:
Yes, copy/paste from AKS inasmuch as is possible.  The only valid scenarios for us are:
- no identity
- system identity
- one user identity, user provides its resource ID

Some resources allow system + user at the same time, or multiple user identities at the same time.  This is not allowed by AKS managed clusters or Fleet at this time.

Reply:
Updated parameters, validators, tests, history. Ready for another review.

Reply:
Addressing test failures...